conda_build/build.py

```diff
@@ -272,6 +553,51 @@ def rewrite_file_with_new_prefix(path, data, old_prefix, new_prefix):
     return data
 
 
+def perform_replacements(matches, prefix):
+    for file, match in matches.items():
+        filename = os.path.join(prefix, file)
+        filename_tmp = filename+'.cbpatch.tmp'
+        if os.path.exists(filename_tmp):
+            os.unlink()
+        shutil.copy2(filename, filename_tmp)
+        print("Patching: {} in {} {}".format(filename,
+                                             (match['submatches']),
+                                             'places' if len(match['submatches']) > 1 else 'place'))
+        with open(filename_tmp, 'wb+') as file_tmp:
+            file_tmp.truncate()
+            with open(filename, 'rb') as file:
+                last_index = 0
+                for submatch in match['submatches']:
+                    length = submatch['start'] - last_index
+                    data = file.read(length)
+                    assert len(data) == length
+                    file_tmp.write(data)
+                    original = submatch['text']
+                    new_string = re.sub(submatch['regex_re'], submatch['replacement_re'], original)
```

Comment:
This allows for use of groups from `regex_re` to be used in `replacements_re` for people needing to do more fancy replacements.